app.js

```diff
@@ -62,6 +62,133 @@ VIEWER.isJSON = function(obj) {
     return r
 }
 
+
+VIEWER.navplaceObject = (function(geojson, navplaces, depth) {
+    //make sure value is a dict and not null
+    if (typeof geojson !== 'object' || geojson === null) {
+        return undefined;
+    }
+    //find navplace in the current object
+    if (geojson.hasOwnProperty('navPlace')) {
+        navplaces.push(geojson['navPlace'])
+    }
+    
+    if (depth > 5) {
+        console.warn("Maximum JSON search depth reached.")
+        return undefined;
+    }
+    //recursive
+    for (var key in geojson) {
+        var value = geojson[key];
+
+        if (Array.isArray(value)) {
+            for (var i = 0; i < value.length; i++) {
+                if (typeof value[i] === 'object' || Array.isArray(value[i])) {
+                    this.navplaceObject(value[i], navplaces, depth+1);
+                }
+            }
+        } else if (typeof value === 'object') {
+            this.navplaceObject(value, navplaces, depth+1);
+        }
+    }
+    return navplaces.filter(item => item !== undefined);
+});
+
+VIEWER.getBbox = (function(navplaceObj){    
+    if ((navplaceObj['type'] == 'FeatureCollection')) {        
+        var features = navplaceObj['features'];
```

Comment:
instead of `var` use `const` or `let`.  I know there are some `var` variable declarations around but we should not use that.

`const` if it doesn't change
`let` if it does